src/server/identity/cmds/cmds_test.go

```diff
@@ -60,7 +60,7 @@ func TestClientCRUD(t *testing.T) {
 		  | match '  - https://localhost:1234' \
                   | match '  - https://localhost:5678' \
 		  | match '  - x' \
-		  | match '  - y' \
+		  | match '  - "?y"?' \
```

Comment:
This is a fix that had been merged upstream in gopkg.in/yaml.v3, that was not in our fork (in yaml, "y" is a bool so it has to be quoted)